app/src/AboutUs/carrierquotes.js

```diff
@@ -0,0 +1,102 @@
+import * as React from "react";
+import "./carousel.css";
+
+const CarrierQuotes = () => {
+  return (
+    <div>
+      <h2>What our carriers and shippers say.....</h2>
+      <div
+        id="myCarousel"
+        class="carousel slide text-center"
+        data-ride="carousel"
+      >
+        {/* <!-- Indicators --> */}
+        <ol class="carousel-indicators">
+          <li data-target="#myCarousel" data-slide-to="0" class="active"></li>
+          <li data-target="#myCarousel" data-slide-to="1"></li>
+          <li data-target="#myCarousel" data-slide-to="2"></li>
+        </ol>
+
+        {/* <!-- Wrapper for slides --> */}
+        <div class="carousel-inner" role="listbox">
+          <div class="carousel-item active">
+            <h4>
+              "I am usually in the dark when it comes to where to take rejected
+              product. I've taken hours in the past just to find a local dump.
+              Thankfully I'm able to find a registered location and find it
+              quickly."
+              <br />
+              <span>Jan Stewart, Rusty's Trucking</span>
```

Comment:
Love how the quotes include individual name and company  to make it more personal! 👍